### ISSUE TITLE
I want to be able to get user feedback on dugga

### ISSUE THREAD

**Issue by a97marbr** [HGustavs/LenaSYS#1934]
When submitting we could add a question.  Did you like the dugga. Or similar? 
How likely are you to propose this course to others?

**Comment by a97marbr** [comment]
Can we incorporate https://en.wikipedia.org/wiki/Net_Promoter thinking ...

**Comment by a14jaksi** [comment]
Just wanted to clarify some details of this feature. 

Is it to be anonymous or not? During a meeting with @HGustavs my group leader (and present group members) understood that each feedback entry would generate a forum thread and be visible in that thread. Despite the forum thread only being visible to the teacher and student I question if this will provide the truest results. 

One pro with anonymous feedback is that you'll surely get a truer representation of the general opinion on the dugga/course (swedes are notorious for not wanting to openly challenge the status quo). Through graphs and other forms of data-representation a teacher could then analyse how changes to a dugga affect general opinion. 

A pro with non-anonymous feedback is that you could have a dialogue with the student who provided the negative feedback and directly request additional information (as in the Net Promoter example).

I think the pros with anonymous feedback outweigh the cons, and I was a bit surprised when I heard the feedback was not to be anonymous. I just wanted to make sure I understood the request correctly.

**Comment by d14alehe** [comment]
Another issue that I would like to be clarified is whether the feedback is going to be based on people adding a comment, if it's gonna be purely score based e.g. between 1 - 10 or a hybrid with both of the previously mentioned features.

There seems to be a discrepancy between what @a97marbr implied at the first meeting when the course was first introduced to us students and what @HGustavs said at our last meeting.

@a97marbr implied that it should be kept as simple as possible, and that the students would just avoid using said feature if it's too time consuming, and used "writing a comment" as an example of what to NOT do.

What @HGustavs implied at the meeting seems to be completely opposite, where the students add a comment which would be generated into a private forum thread where the involved parties can discuss it further. Which in my opinion seems very time consuming for the students, as well as the fact that we won't be able to generate valuable data based on pure text entries as @a14jaksi mentioned above.

I would be very much obliged if there would be some clarifications on which of these "paths" is the right one before we start addressing this issue.

Thanks.

**Comment by a97marbr** [comment]
Seldom is the world black or white ...

From my perspective I would like a simple measurement. However, for this to be meaningful it should be possible to connect with students who are unhappy with a dugga and/or course. So that we can ask if they want to share what wasn't working for them. It could be as simple as an "are we allowed to contact you?" question.

I am not against the private thread thing, but I would argue that it is optional and initiated after a student has agreed to contact. 

The work flow something like:
Student say '3' and agrees to being contacted. -> A teacher sees the score and that he is allowed to contact the student. -> Contact is made (telephon/email/coffee/whaterever) -> A forum thread is started based on this contact

If the student on the other hand say '3' and do not agree to being contacted, the score is added but no connection to which student is made. Thus anonymity is kept.

**Comment by a18viksa** [comment]
Closing issue after merge to master.
